test/quick_test/test_errors_in_logfiles.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import yaml
+from pathlib import Path
+
+from .common import *
+
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+def load_errors():
+    """
+    Load the errors and whitelist strings from ./data/logfile.errors.yaml.
+
+    Returns
+    -------
+    errors : list[str]
+        List of errors.
+    whitelist : list[str]
+        List of whitelisted errors.
+    """
+
+    # Open the error_file_path and read the errors and whitelist into arrays
+
+    error_file_path = Path(__file__).parent / "data" / "logfile.errors.yaml"
+
+    with open(error_file_path, "r") as file:
+        data = yaml.safe_load(file)
+        errors = data.get("errors", [])
+        whitelist = data.get("whitelist", [])
+
+    return errors, whitelist
+
+
+def load_log_files(test_subject: Path):
+    """
+    Retrieve the log files in the given log directory.
+
+    Parameters
+    ----------
+    test_subject : Path
+        Subject directory to test.
+
+    Returns
+    -------
+    log_files : list[Path]
+        List of log files in the given log directory.
+    """
+
+    # Retrieve the log files in given log directory
+
+    log_directory = test_subject / "scripts"
+    log_files = [file for file in Path(log_directory).iterdir() if file.suffix == ".log"]
+
+    return log_files
+
+
+@pytest.mark.parametrize("test_subject", load_test_subjects())
+def test_errors(subjects_dir: Path, test_dir: Path, test_subject: Path):
+    """
+    Test if there are any errors in the log files.
+
```

Comment:
Add documentation on the fact that subjects_dir and test_dir are filled by pytest fixtures defined in `conftest`

Reply:
done